extensions/wikia/TemplateClassification/specials/TemplatesSpecialController.class.php

```diff
@@ -0,0 +1,335 @@
+<?php
+
+use Wikia\ContentReview\ContentReviewStatusesService;
+use Wikia\ContentReview\Helper;
+
+class TemplatesSpecialController extends WikiaSpecialPageController {
+
+	const ITEMS_PER_PAGE = 20;
+
+	public $type;
+	public $templateName;
+	public $templates = [];
+	public $groups;
+
+	function __construct() {
+		parent::__construct( 'Templates' );
+	}
+
+	public function init() {
+		$this->specialPage->setHeaders();
+		$this->specialPage->getOutput()->setPageTitle( wfMessage( 'special-templates' )->escaped() );
```

Comment:
`OutputPage::setPageTitle` escapes the title passed to it, so the output format can be `->plain()`.